jme3-core/src/main/resources/Common/MatDefs/Light/PBRLighting.frag

```diff
@@ -1,3 +1,5 @@
+#extension GL_NV_shadow_samplers_cube : enable
+
```

Comment:
Why is this needed?
1. this is a NVidia only extension I wonder why it's in a fix for AMD cards
2. It's an extention that allows you to fetch into cube shadow maps. Things we don't do at all, and if we did it wouldn't be in this shader.